src/bundle/Resources/public/js/scripts/admin.table.js

```diff
@@ -97,6 +97,28 @@
 
         tablesCheckboxesChangeListeners.delete(table);
     };
+    let prevShouldShowRightColumnShadow = false;
```

Comment:
This variable should be one per table, not one for all tables.
But, actually, do we need`shouldShowRightColumnShadow !== prevShouldShowRightColumnShadow` check?